openapi/openapi.yaml

```diff
@@ -1084,15 +1084,20 @@ components:
             $ref: "#/components/schemas/AnalysisResult"
       required: [analysis_results]
 
+
+
     SystemAnalysesReturn:
       type: object
       properties:
         system_analyses:
           type: array
           items:
             $ref: "#/components/schemas/SingleAnalysis"
+        significance_test_info:
+          type: object    
```

Comment:
Could you please define a data type for this? I know that just using "object" is easier in the short term, but in terms of long term readability and robustness I think it's a lot better to know the fields that are expected in the object.

Reply:
fixed